testimony/__init__.py

```diff
@@ -63,26 +57,17 @@ def prefixed_lines():
 
 
 def is_test_module(filename):
-    """Return ``True`` if the ``filename`` matches an expected test
-    module name.
-    """
+    """Indicate if ``filename`` match a test module file name."""
```

Comment:
A good practice is to use on imperative mode. I'm following it.

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

From https://www.python.org/dev/peps/pep-0257/